packages/client/components/JiraScopingSearchResults.tsx

```diff
@@ -72,25 +93,56 @@ const JiraScopingSearchResults = (props: Props) => {
     })
     const isQueryNew = !searchHashes.includes(lookupKey)
     if (isQueryNew) {
-      PersistJiraSearchQueryMutation(atmosphere, {teamId, input: {queryString, isJQL, projectKeyFilters: projectKeyFilters as string[]}})
+      PersistJiraSearchQueryMutation(atmosphere, {
+        teamId,
+        input: {queryString, isJQL, projectKeyFilters: projectKeyFilters as string[]}
+      })
     }
   }
   return (
     <>
-      <JiraScopingSelectAllIssues usedJiraIssueIds={usedJiraIssueIds} issues={edges} meetingId={meetingId} />
-      <ResultScroller>
-        {edges.map(({node}) => {
-          return <JiraScopingSearchResultItem key={node.id} issue={node} isSelected={usedJiraIssueIds.has(node.id)} meetingId={meetingId} persistQuery={persistQuery} />
-        })}
-      </ResultScroller>
+      {edges.length === 0 && !isEditing ? (
+        <JiraScopingNoResults error={error?.message} />
+      ) : (
+        <>
+          <JiraScopingSelectAllIssues
+            usedJiraIssueIds={usedJiraIssueIds}
+            issues={edges}
+            meetingId={meetingId}
+          />
+          <ResultScroller>
+            <NewJiraIssueInput
+              isEditing={isEditing}
+              meeting={meeting}
+              setIsEditing={setIsEditing}
+              viewer={viewer}
+            />
+            {edges.map(({node}) => {
+              return (
+                <JiraScopingSearchResultItem
+                  key={node.id}
+                  issue={node}
+                  isSelected={usedJiraIssueIds.has(node.id)}
+                  meetingId={meetingId}
+                  persistQuery={persistQuery}
+                />
+              )
+            })}
+          </ResultScroller>
+        </>
+      )}
+      <Button onClick={() => setIsEditing(true)} palette='blue'>
```

Comment:
i think we should make this button hidden when you're editing a new issue.
alternatively, clicking it again should re-focus the new issue.